src/current/cockroachcloud/production-checklist.md

```diff
@@ -6,43 +6,127 @@ cloud: true
+
+## SQL Best Practices
 
 To ensure optimal SQL performance for your CockroachDB {{ site.data.products.cloud }} cluster, follow the best practices described in the [SQL Performance Best Practices](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/performance-best-practices-overview) guide.
 
 ## Use a pool of persistent connections
 
 Creating the appropriate size pool of connections is critical to gaining maximum performance in an application. Too few connections in the pool will result in high latency as each operation waits for a connection to open up. But adding too many connections to the pool can also result in high latency as each connection thread is being run in parallel by the system. The time it takes for many threads to complete in parallel is typically higher than the time it takes a smaller number of threads to run sequentially.
 
-For guidance on sizing, validating, and using connection pools with CockroachDB, see [Use Connection Pools](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/connection-pooling).
+For guidance on sizing, validating, and using connection pools with CockroachDB, refer [Use Connection Pools](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/connection-pooling).
 
 ## Keeping connections current
 
 After an application establishes a connection to CockroachDB {{ site.data.products.cloud }}, those connections can occasionally become invalid. This could be due to changes in the cluster topography, rolling [upgrades]({% link cockroachcloud/upgrade-policy.md %}) and restarts, network disruptions, or cloud infrastructure unavailability.
 
-Make sure connection validation and retry logic is used by your application. Validating and retrying connections is typically handled by the driver, framework, or the connection pool used by an application. For guidance on connection pool sizing, connection validation, and connection retry logic, see [Use Connection Pools](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/connection-pooling).
+Set the maximum lifetime of a connection to between 5 and 30 minutes. {{ site.data.products.dedicated }} and {{ site.data.products.serverless }} support 30 minutes as the maximum connection lifetime. When a node is shut down or restarted, client connections can be reset after 30 minutes, causing a disruption to applications.
 
-## Transaction retries
+Ensure that your application can handle disruptions by implementing connection validation and [retry logic](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/transaction-retry-error-reference) appropriately. Cockroach Labs recommends using connection pools to manage the lifecycle of client connections to application servers. For guidance on connection pool sizing, connection validation, and connection retry logic, refer to [Use Connection Pools](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/connection-pooling).
```

Comment:
we no longer recommend "connection retry logic" as of https://github.com/cockroachdb/docs/pull/17956

I think this paragraph is mixing up two different things:
- transaction retry logic is still needed. It's used when CRDB returns a serialization error (an error with a code of 40001). This is described here: https://www.cockroachlabs.com/docs/stable/transaction-retry-error-reference
- connection retry logic is something we used to recommend so an app could handle a CRDB node shutting down while the app was still connected to it. We no longer recommend that, and instead recommend using appropriate connection pool settings now.

Reply:
@rafiss Can I delete the paragraph since we talk about using a pool of connections and transaction retries in other sections?

So the content for this section would be:


## Keeping connections current

After an application establishes a connection to CockroachDB {{ site.data.products.cloud }}, those connections can occasionally become invalid. This could be due to changes in the cluster topography, rolling [upgrades]({% link cockroachcloud/upgrade-policy.md %}) and restarts, network disruptions, or cloud infrastructure unavailability.

Set the maximum lifetime of a connection to between 5 and 30 minutes. {{ site.data.products.dedicated }} and {{ site.data.products.serverless }} support 30 minutes as the maximum connection lifetime. When a node is shut down or restarted, client connections can be reset after 30 minutes, causing a disruption to applications.

Reply:
that new paragraph looks good to me!

Reply:
@rafiss Updated the content. Can you verify the change and approve the PR?